### PR TITLE
Fixed typos in http status code constants docblocks

### PR DESCRIPTION
**Pull request by othercorey**
Missed typo in https://github.com/cakephp/cakephp/pull/13785.

### CODE REVIEW

File: src/Http/Response.php [hunk 1]
```diff
@@ -35,7 +35,7 @@
 /**
  * Responses contain the response text, status and headers of a HTTP response.
  *
- * There are external packages such as `http-message-util` that provide HTTP
+ * There are external packages such as `fig/http-message-util` that provide HTTP
```
Comment (ADmad) [inline src/Http/Response.php]: Should be `psr` not `fig`.

Comment (othercorey) [reply src/Http/Response.php]: Is it?  I thought I pulled the official name.  I'll double-check.

Comment (othercorey) [reply src/Http/Response.php]: It is fig/http-message-util for the constants not the interface.  Am I missing something?

Comment (ADmad) [reply src/Http/Response.php]: My mistake, seems they use `psr` only for standards packages and `fig` for others.

Comment (dereuromark) [reply src/Http/Response.php]: Quite the bad behavior and antipattern to not match the URL:
https://github.com/php-fig/http-message-util

The correct naming would IMO be `php-fig/http-message-util` - but meeeh

**Pull request merged**